### PR TITLE
Upgrade to Ember 3

### PR DESCRIPTION
**Pull request by elwayman02**

### CODE REVIEW

File: README.md
Comment (acorncom) [inline README.md]: Looks like this icon went missing in the upgrade

Comment (elwayman02) [reply README.md]: Apologies, that should be fixed now! For future reference, I think project maintainers have access to edit PRs now when minor changes like this are needed. Sorry for the delay in getting back to this.

**Comment by elwayman02** [comment]
@acorncom I'm not sure why the build is showing as failed. If you click into the travis builds, it actually shows all tests as succeeding and then the build exits.

**Comment by acorncom** [comment]
@elwayman02 Hmm, weird. I'm sadly not a maintainer on this repo, @michalsnik do you have any thoughts?

**Comment by michalsnik** [comment]
Hey @elwayman02 and @acorncom :)

Thanks for working on this! It looks like `The command "yarn lint:js" exited with 1.`

File: .travis.yml
Comment (michalsnik) [inline .travis.yml]: I'd also add latest LTS version of Node, just to be sure :)

File: README.md
Comment (michalsnik) [inline README.md]: Lets add 🍻 here

**Comment by elwayman02** [comment]
@michalsnik should be good to go now! Thanks for the review :)

**Comment by yohanmishkin** [comment]
Awesome, looking forward to this!

**Comment by RuslanZavacky** [comment]
Yay, waiting for this as well :)

**Comment by elwayman02** [comment]
@michalsnik do you have any remaining concerns I should address before you can merge this PR?

**Review by michalsnik — APPROVED** [review]
Thank you @elwayman02 ! LGTM 🚀

**Comment by michalsnik** [comment]
`v1.0.0` released. Enjoy and sorry for the delay guys :)

**Pull request merged**
